### PR TITLE
dlt_dbt_runner_article_blog

### PR DESCRIPTION
**Pull request by dat-a-man**
### Related Issues

- Fixes #...
- Closes #...
- Resolves #...

### CODE REVIEW

File: docs/website/blog/2024-01-12-dlt-dbt-runner-on-cloud-functions.md [hunk 1]
```diff
@@ -313,13 +313,13 @@ took another 1h per day, we would need to pay ~15 USD/month for the smallest mac
 can see how that would be expensive if we wanted to run it continuously or had multiple pipelines always-on in parallel.
 
 Cloud functions are serverless lightweight computing solutions that can handle small computational
-workloads and are cost-effective. dbt doesn't require the high computing power of the machine
+workloads and are cost-effective. DBT doesn't require the high computing power of the machine
```
Comment (AstrakhantsevaAA) [inline docs/website/blog/2024-01-12-dlt-dbt-runner-on-cloud-functions.md:316]: why it's uppercase?

**Review by AstrakhantsevaAA — CHANGES_REQUESTED** [review]
could you also fix code snippets, use python type for all snippets and break lines

**Comment by AstrakhantsevaAA** [comment]
image
it is not readable

File: docs/website/blog/2024-01-12-dlt-dbt-runner-on-cloud-functions.md (hunk 1 above)
Comment (dat-a-man) [reply docs/website/blog/2024-01-12-dlt-dbt-runner-on-cloud-functions.md]: done

**Pull request closed without merging**